pkgs/development/interpreters/python/cpython/3.7/default.nix

```diff
@@ -145,7 +145,11 @@ in stdenv.mkDerivation {
     # some $TMPDIR references to improve binary reproducibility.
     # Note that the .pyc file of _sysconfigdata.py should be regenerated!
     for i in $out/lib/python${majorVersion}/_sysconfigdata*.py $out/lib/python${majorVersion}/config-${majorVersion}m*/Makefile; do
-      sed -i $i -e "s|-I/nix/store/[^ ']*||g" -e "s|-L/nix/store/[^ ']*||g" -e "s|$TMPDIR|/no-such-path|g"
+      sed -i $i \
+        -e "s|-I/nix/store/[^ ']*||g" \
```

Comment:
I know this is an existing problem, but we shouldn't assume that the Nix store is at `/nix/store`.

Reply:
`NIX_BUILD_TOP` is the best thing to use instead?

Reply:
I did some digging, and it looks like `nuke-references` uses `$NIX_STORE`; I updated the PR to use that instead!